java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -297,6 +299,21 @@ public CompletableFuture<Long> hdel(@NonNull String key, @NonNull String[] field
         return commandManager.submitNewCommand(HashDel, args, this::handleLongResponse);
     }
 
+    @Override
+    public CompletableFuture<Long> hincrBy(@NonNull String key, @NonNull String field, long amount) {
```

Comment:
we are not using @ NonNull to `amount` because it's long and not Long?

Reply:
Yes! The idea behind using primitives vs nullables is that we can pass in numbers as 1 instead of 1L which is an easier interface for the user. (conversion is done under the hood via java)